src/components/LayerUI.tsx

```diff
@@ -104,6 +99,17 @@ const DefaultMainMenu: React.FC<{
   );
 };
 
+type Tunnel = ReturnType<typeof tunnel>;
+export const TunnelsContext = React.createContext<{
+  mainMenuTunnel: Tunnel;
+  welcomeScreenMenuHintTunnel: Tunnel;
+  welcomeScreenToolbarHintTunnel: Tunnel;
+  welcomeScreenHelpHintTunnel: Tunnel;
+  welcomeScreenCenterTunnel: Tunnel;
+  footerCenterTunnel: Tunnel;
+  jotaiScope: symbol;
+}>(null!);
```

Comment:
can there be a scenario where the context is used before it's initialized?

Reply:
Ummm I don't think so that would ever happen

Reply:
we can remove the word `Tunnel` from all the keys since its a tunnel object

Reply:
I did that initially but it doesn't work well unless we stop destructing